masterlist.yaml

```diff
@@ -3029,6 +3029,27 @@ plugins:
       - crc: 0x7B69E509
         util: 'SSEEdit v4.0.4b'
 
+  - name: '(PSI|Thunderbolt fix test)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/83935/'
+        name: 'Parallax Spell Impacts'
+  - name: 'PSI.esp'
+    req: [ *ENBSeries ]
+    msg:
+      - <<: *includesX
+        subs: [ '[Deadly Spell Impacts Transparency Fix for ENB](https://www.nexusmods.com/skyrimspecialedition/mods/41284/)' ]
+        condition: 'active("DeadlySpellImpacts Transparency Fix.esp")'
```

Comment:
In cases where a plugin can be wholly removed because it's included or otherwise implemented in another plugin, I'd prefer to add `alreadyInX` to it. `includesX` is more for if you might still have a reason to keep it around (e.g., you might still want to install Modern Brawl Bug Fix separately to make sure its scripts win any BSA asset conflicts).